music21/musicxml/m21ToXml.py

```diff
@@ -3265,55 +3280,88 @@ def parseFlatElements(
 
         self.currentVoiceId = voiceId
 
-        # group all objects by offsets and then do a different order than normal sort.
-        # that way chord symbols and other 0-width objects appear before notes as much as
+        # We order things differently if there are any SpannerAnchors in the flat stream.
+        # If there are SpannerAnchors, we do two passes. The first pass skips all
+        # prePostObjectSpanners processing, and then the second pass goes back to the
```

Comment:
Ugh.  This phrasing is my fault, I think.  Is there a better phrase than `prePostObjectSpanners` it was okay when it only appeared twice in the code but now that it's all over the place i don't have a clue what that means (same in `skipPrePostObjectSpanners`).  relatedSpanners might be better?  skipRelatedSpanners, etc.

Reply:
done